### ISSUE TITLE
Make more use of "set build name" plugin to identify which builds are which

### ISSUE THREAD

**Issue by sxa555** [AdoptOpenJDK/openjdk-infrastructure#434]
e.g. showing version numbers on build/test jobs
https://wiki.jenkins.io/display/JENKINS/Build+Name+Setter+Plugin is the plugin and we currently have it on the JCK jenkins, but it's not being used on the main one.

**Comment by sxa555** [comment]
You ok with me adding this one @karianna ?

**Comment by karianna** [comment]
Yes, was about to do it and then got side tracked by upgrading existing ones for security.

**Comment by sxa** [comment]
For some reason I'm not seeing this in the web UI. More investigation required. Bumping to ensure that happens.

**Comment by sxa** [comment]
Bumping this since I want to see if we can make properly use of this for things like VPC to identify which run is which without using pipelines.

**Comment by sxa** [comment]
Will aim to do this as part of the restarts required by https://github.com/adoptium/infrastructure/issues/2961

**Comment by sxa** [comment]
Also added to the [download_and_sbom_validation](https://ci.adoptium.net/job/build-scripts/job/release/job/download_and_sbom_validation/) job

**Comment by sxa** [comment]
Noting that we could perhaps also make more use of the "embeddable build status icons" which are already used elsewhere
